Sources/PackageCollections/Providers/GitHubPackageMetadataProvider.swift

```diff
@@ -43,7 +43,7 @@ struct GitHubPackageMetadataProvider: PackageMetadataProvider {
         }
 
         let metadataURL = baseURL
-        let tagsURL = baseURL.appendingPathComponent("tags")
+        let releasesURL = URL(string: baseURL.appendingPathComponent("releases").absoluteString + "?per_page=20") ?? baseURL.appendingPathComponent("releases")
```

Comment:
will this return the last 20 releases?

Reply:
Yes, it returns releases in reverse chronological order.

Reply:
Using `URLComponents` to construct the `URL` is probably better but feels overkill for one simple query param.

Reply:
the construction of the URL seems fine to me. the one question is if 20 most recent releases is enough, or we need more

Reply:
We can increase it to 50?. The API's limit is 100.

Reply:
right, I am not sure how to determine what the right number should be. I guess this information is supplemental and optional in any case. is that correct?

Reply:
> I guess this information is supplemental and optional in any case. is that correct?

Yes, and it's best-effort trying to match these with versions in the collections--it's possible for there not be no overlap at all.

Reply:
I think then 20 should be fine